ionic4-angular-example/README.md

```diff
@@ -12,14 +12,14 @@ $ ionic cordova plugin add cordova-plugin-one
 
```

Comment:
please ensure the feedback from yesterday's session is also added to the 'old' example app readmes

Reply:
Updated